### ISSUE TITLE
Mobile Geolocation in Google Analytics seems Off

### ISSUE THREAD

**Issue by oxtoacart** [getlantern/lantern#4092]
cc: @atavism 

A lot of users are appearing in the locations of their proxies (Amsterdam, New York, Japan).

![image](https://cloud.githubusercontent.com/assets/5000654/14824492/d813f36e-0b9b-11e6-84f4-1fd1cd05442b.png)

**Comment by oxtoacart** [comment]
Making this a P2 in case it indicates a larger problem with proxying.  Likely it's related to us being the full device VPN and the Google Analytics requests going through that.

**Comment by myleshorton** [comment]
Yeah we don't do anything on mobile to tell analytics our real IP @oxtoacart, so it just uses the proxy IPs.

**Comment by myleshorton** [comment]
It is very odd we see so many in Amsterdam, though. Users being assigned incorrectly possibly @aranhoide?
